challenges.html

```diff
@@ -124,6 +124,29 @@ <h1 class="ourChallengesHeader">Our challenges</h1>
         </p>
         <input type="text" id="userInput" placeholder="Minimum 3 characters">
       </div>
+      <div class="sortFilter">
+        <p>
+          Sort challenges
+        </p>
+        <div class="sortButtons">
+          <label>
+            <input type="radio" id="sortTitleAlphabet" name="sortFilterBtn">
+            Title (A-Z)
+          </label>
+          <label>
+            <input type="radio" id="sortMaxRating" name="sortFilterBtn">
+            Highest to lowest (5-0)
+          </label>
+          <label>
+            <input type="radio" id="sortMinRating" name="sortFilterBtn">
```

Comment:
Maybe change to type checkbox instead of type radio better match the **by type** filter better

Reply:
Yes, but the problem with checkboxes is that you can have multiple choices selected and since sorting is only one choice _(can't sort min to max rating and max to min rating at the same time)_. I could instead try to style the boxes to **look** like checkboxes if that is what you want.

Reply:
No that seems a lot of work for such a little thing. Go with radio 👍